### PR TITLE
[MINOR][TEST][DOC] Execute action miss name message

### PR DESCRIPTION
**Pull request by uncleGen**
## What changes were proposed in this pull request?

some minor updates:
- `Execute` action miss `name` message
-  typo in SS document
-  typo in SQLConf

## How was this patch tested?

N/A

### CODE REVIEW

**Comment by uncleGen** [comment]
retest this please

**Comment by uncleGen** [comment]
cc @cloud-fan

File: sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala [hunk 1]
```diff
@@ -294,7 +294,7 @@ trait StreamTest extends QueryTest with SharedSQLContext with TimeLimits with Be
   /** Execute arbitrary code */
   object Execute {
     def apply(name: String)(func: StreamExecution => Any): AssertOnQuery =
-      AssertOnQuery(query => { func(query); true }, "name")
+      AssertOnQuery(query => { func(query); true }, name)
```
Comment (srowen) [inline sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamTest.scala]: CC @tdas for a double check but looks like a correct fix

**Review by srowen — APPROVED** [review]

**Review by attilapiros — APPROVED** [review]
LGTM

**Comment by tdas** [comment]
Lgtm.

**Comment by cloud-fan** [comment]
thanks, merging to master!

**Pull request closed without merging**